definitions/npm/koa-logger_v3.2.x/flow_v0.53.x-/koa-logger_v3.2.x.js

```diff
@@ -0,0 +1,15 @@
+declare module 'koa-logger' {
+  declare type TransporterArguments =
+    // This argument can take these 2 forms:
+    // - Request: [format, method, url]
+    // - Response: [format, method, url, status, time, length]
+    // Typing it with only 1 tuple instead of a union makes it easier to work with.
```

Comment:
or so I think... I don't use this myself so maybe future users will decide otherwise...